### PR TITLE
Add project from Toronto Hub 2023 to website

### PR DESCRIPTION
**Pull request by sunclara**
Changes proposed in this pull request:

- Add project titled "fMRI_ADHD_connectivity" to the Brainhack school website 
- Almost all commits are edits to the "index.md" file in the "fMRI_ADHD_connectivity" project folder
- All other commits are image uploads
- Please find the index.md file at this path: sunclara/brainhack_project_page/content/en/project/fMRI_ADHD_connectivity/index.md

### CODE REVIEW

File: content/en/project/fMRI_ADHD_connectivity/index.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+# Please only lowercase letters
+tags: [project, github, markdown, brainhack, jupyter notebook, preprocessing, ADHD, fMRI]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects.. <!-- this is changed as well -->
+
+summary: "The focus of our project was to gain experience using neuroimaging tools to preprocess, analyze, and visualize functional MRI data. We aimed to explore differential variability in brain connectivity among children with and without ADHD. Project reports are incorporated on the BHS [website](https://school.brainhackmtl.org/project)."
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: "bhs2020.png"
```
Comment (pbellec) [inline content/en/project/fMRI_ADHD_connectivity/index.md:26]: Please add a custom cover image for your project.

**Review by pbellec — CHANGES_REQUESTED** [review]
I've rendered the page and everything looks great. Congrats on the awesome project! I'll wait a bit for review of content by the Toronto hub, but once you address the minor issue I raised I'm ready to merge.

File: content/en/project/fMRI_ADHD_connectivity/index.md (hunk 1 above)
Comment (sunclara) [reply content/en/project/fMRI_ADHD_connectivity/index.md]: Thank you for the feedback and suggestion Pierre! We have now added a custom cover image. Great to see so many incredible projects starting to show up on the Brainhack School website!

**Review by juchiyu — APPROVED** [review]

**Review by pbellec — APPROVED** [review]

**Comment by pbellec** [comment]
Sorry this took so long, merging.

**Pull request merged**